students/K_Shaffer/mailroom/mailroom02.py

```diff
@@ -0,0 +1,60 @@
+'''
+Script: mailroom
+Date: 10/28/2019
+Dev: Kory Shaffer
+'''
+
+
+donors = {'William Gates, III': (653784.49, 2),  'Mark Zuckerberg': (16396.10, 3), 'Jeff Bezos': (877.33, 1),
+          'Paul Allen': (708.42, 3)}
+
+
+def main(donors):
+    user_input(donors)
```

Comment:
If you run option 2 after option 1, the code breaks:

Donor Name          | Total Given | Num Gifts | Average Gift 
------------------------------------------------------------
William Gates, III   $   653784.49          2  $   326892.24
Mark Zuckerberg      $    16396.10          3  $     5465.37
Traceback (most recent call last):

  File "<ipython-input-27-d4fa85b70326>", line 64, in <module>
    main(donors)

  File "<ipython-input-27-d4fa85b70326>", line 13, in main
    user_input(donors)

  File "<ipython-input-27-d4fa85b70326>", line 27, in user_input
    create_report(donors)

  File "<ipython-input-27-d4fa85b70326>", line 57, in create_report
    print('{:<20} ${:>12.2f}{:>11}  ${:>12.2f}'.format(donor, donors[donor][0], donors[donor][1],

KeyError: 0